.github/workflows/regular-release.yaml

```diff
@@ -0,0 +1,66 @@
+on:
+  schedule:
+#            ┌───────────── minute (0 - 59)
+#            │ ┌───────────── hour (0 - 23)
+#            │ │ ┌───────────── day of the month (1 - 31)
+#            │ │ │ ┌───────────── month (1 - 12 or JAN-DEC)
+#            │ │ │ │ ┌───────────── day of the week (0 - 6 or SUN-SAT)
+#            │ │ │ │ │
+#            │ │ │ │ │
+#            │ │ │ │ │
+    - cron: '0 9 * * THU'
+      # ^ this means 9:00 AM every thursday
+      # I can’t figure out the right syntax
+      # for ‘every other thursday’
+
+jobs:
+  bump_cbmc_version:
+    runs-on: ubuntu-20.04
+    outputs:
+      cbmc_version: ${{ steps.cbmc_version_number.outputs.CBMC_VERSION }}
+      bump_git_sha: ${{ steps.commit_bump.outputs.bump_git_sha }}
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          submodules: recursive
+      - name: Get new CBMC version number
+        id: cbmc_version_number
+        run: |
+          NEW_CBMC_VERSION=$(grep '^CBMC_VERSION =' src/config.inc | cut -d = -f 2 | scripts/increment_version.sh)
+          echo ::set-env name=CBMC_VERSION::$NEW_CBMC_VERSION
+          echo ::set-output name=CBMC_VERSION::$NEW_CBMC_VERSION
+      - name: Update CBMC version
+        run: |
+          sed -i "s/CBMC_VERSION = .*/CBMC_VERSION = $CBMC_VERSION/" src/config.inc
+      - name: Set git identity to github bot
+        run: |
+          git config --local user.name db-ci-cprover
+          git config --local user.email "db-ci-cprover@diffblue.com"
+      - name: Commit changes
+        id: commit_bump
+        run: |
+          git commit -a -m "Bump version to $CBMC_VERSION"
+          echo "::set-output name=bump_git_sha::$(git rev-parse HEAD)"
+      - name: Push changes
+        run: |
+          git push
```

Comment:
:grey_question: Which branch is this pushing to? Might be nice to have a comment with the answer.

Reply:
We didn’t specify a branch in `checkout`, so we pull from the default (`develop`), which automatically also sets the upstream of the branch to `develop`. So this will push against develop.